scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -20,9 +20,9 @@
       name: SLES12SP3-Updates
 
   # Devel:Cloud:8
-  - name: Add Devel:Cloud:8:Staging media
+  - name: "Add {{ cloudsource }} media"
```

Comment:
Can you add a default value for this new variable in this file? That way, repository.yml can be used without the ```hosts``` file (which is generated by the jenkins job)